plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/js/WSDLOperations.js

```diff
@@ -0,0 +1,292 @@
+/*
+ * Signature: CreateFile(currentTaskName){...}
+ * 
+ * This method generates WSDL files related to the Tasks. It copies content from
+ * the dummy wsdl file in the project and creates new wsdl with the intended
+ * name. Then a DOM from the created dummy is created and sent to further
+ * processing. The Task Name which is used for generating the wsdl should be
+ * given.
+ * 
+ * 
+ */
+
+
+/*
+ * Signature: saveWSDL(inputWsdl, fileName) {...}
+ * 
+ * This method saves generated wsdl dom with the modifications done in
+ * generateWSDL methods into the WSDL file created. A WSDL dom(inputWSDL) and a
+ * file name(fileName) should be provided as the parameters
+ * 
+ * 
+ */
+function saveWSDL(inputWsdl, fileName) {
+    // Read contents of the file.
+    wsdlContent = new XMLSerializer().serializeToString((inputWsdl));
+    try {
+        IDESaveContentWithExtention(wsdlContent, "", fileName, "wsdl");
+    } catch (err) {
+        alert(err);
+    }
+}
+
+/*
+ * Signature: generateInputWSDL(wsdlDom, currentTaskName) {...}
+ * 
+ * This method fills the empty wsdl DOM created for the new WSDL file with the
+ * respective data. The data is acquired from the elements in the UI editor. The
+ * initial WSDL DOM from the dummy wsdl(wsdlDom) and the current task
+ * name(currentTaskName) should be provided.
+ * 
+ */
+function generateInputWSDL(wsdlDom, currentTask) {
+    currentTaskName = currentTask + "wrapper";
+    // Get internal parameters
+    serviceURL = $('#' + currentTaskName + ' #taskCallbackServiceURL').val();
+    operationName = $('#' + currentTaskName + ' #taskCallbackOperationName')
+        .val();
+    serviceName = $('#' + currentTaskName + ' #taskCallbackServiceName').val();
+    portName = currentTask + "Port";
+    inputElements = $('#' + currentTaskName + ' #inputmappingTable tr');
+
+    // get Definitions node and Schema node from dummy wsdl dom
+    wsdlSchema = wsdlDom.getElementsByTagName("schema")[0];
+    wsdlDefinitions = wsdlDom.getElementsByTagName("definitions")[0];
+
+    // create Data type elements
+    newComplexType = wsdlDom.createElementNS(
+        "http://www.w3.org/2001/XMLSchema", "xsd:complexType");
+    newComplexType.setAttribute("name", currentTask + "DataType"); // Data type
+    // Name
+    newSequence = wsdlDom.createElementNS("http://www.w3.org/2001/XMLSchema",
```

Comment:
"http://www.w3.org/2001/XMLSchema"
Following string has been used in many places over the script. It's better to create a constant variable and use it where necessary.